Framework/API/inc/MantidAPI/MultipleFileProperty.h

```diff
@@ -119,7 +119,12 @@ namespace API {
 class DLLExport MultipleFileProperty
     : public Kernel::PropertyWithValue<std::vector<std::vector<std::string>>> {
 public:
-  /// Constructor
+  /// Default constructor
```

Comment:
This is not a default constructor? Also, Doxygen for member methods should be in the source, not in the header (actually you have them in the source, so just remove those from the header).

The same goes for other methods you added to the header file below.